feature/gnoi/factory_reset/factory_reset_test.go

```diff
@@ -0,0 +1,206 @@
+package factory_reset
+
+import (
+	"context"
+	"fmt"
+	"path"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/openconfig/featureprofiles/internal/fptest"
+	frpb "github.com/openconfig/gnoi/factory_reset"
```

Comment:
can you please move frpb "github.com/openconfig/gnoi/factory_reset" at the bottom of the list?